### PR TITLE
ADD 3,4,5,6 tests

### PR DESCRIPTION
**Pull request by AndrSam1**

### CODE REVIEW

**Comment by AndrSam1** [comment]
[AT_11.03.02_03 | Educations > Menu item [Day Trading] > Test button [Start trading] on Main banner](https://trello.com/c/0Uubs6Pf/108-at11030203-educations-menu-item-day-trading-test-button-start-trading-on-main-banner)

**Comment by AndrSam1** [comment]
[AT_11.03.02_04 | Educations > Menu item [Day Trading] > Test button [Try Demo] on Main banner](https://trello.com/c/FpPpGqzU/115-at11030204-educations-menu-item-day-trading-test-button-try-demo-on-main-banner)

**Comment by AndrSam1** [comment]
[AT_11.03.02_05 | Educations > Menu item [Day Trading] > Test button [Trade] on Widget "Most Traded"](https://trello.com/c/gcoIZyD6/116-at11030205-educations-menu-item-day-trading-test-button-trade-on-widget-most-traded)

**Comment by AndrSam1** [comment]
[AT_11.03.02_06 | Educations > Menu item [Day Trading] > Test button [Start trading] in Content block](https://trello.com/c/TF2ZeyU2/117-at11030206-educations-menu-item-day-trading-test-button-start-trading-in-content-block)

File: conftest.py [hunk 1]
```diff
@@ -223,7 +223,7 @@ def init_remote_driver_chrome():
 
     # !!!
     # если следующую строку раскомментировать, то Chrome отображаться не будет
-    chrome_options.add_argument(conf.CHROMIUM_HEADLESS)
+    # chrome_options.add_argument(conf.CHROMIUM_HEADLESS)
```
Comment (AlexTom22) [inline conftest.py:226]: перед тем как коммитить и пушить, не забывай возвращать безголовый режим браузера

**Review by AlexTom22 — APPROVED** [review]

**Pull request merged**